### DISCUSSION TITLE
textX playground

### DISCUSSION THREAD

**Discussion by igordejanovic** [textX/textX#424] [Announcements]
I'm announcing the availability of [textX playground](https://github.com/textX/textx-playground) where you can play with textX grammars/models.

You can try it out [here](https://textx.github.io/textx-playground/)

This excellent work is done by @AirmiX who also joined textX org and will be maintaining the playground and possibly some other textX projects.

**Comment by goto40** [comment]
Great work. Did you also work on the editor support?

**Reply by igordejanovic** [reply]
If by editor you mean [VS Code plugin](https://github.com/textX/textX-LS) part of textX-LS project, to my knowledge that project is stale for quite some time. @AirmiX took some code from there for the playground but we should ask him are there any improvement that could go back to VS Code plugin. These two are based on a pretty much same tech so I guess that there are parts of the code that could be shared and jointly improved.

**Reply by goto40** [reply]
Very nice, @AirmiX - I am looking forward to see more of your work!

**Reply by AirmiX** [reply]
Thank you @goto40! 🙂 
As @igordejanovic mentioned, there are some improvements that could be taken from the playground and applied to the VS Code plugin and textX-LS project. I plan to work on these in the following months.

**Comment by igordejanovic** [comment]
Be sure not to miss a visualization support of the playground for both grammars and models. It is a little image icon in the upper right of both editors.

<img src="https://github.com/textX/textX/assets/1673425/2ac496dd-fb53-492c-95b9-3f4b5464fd3f" width=30% height=30%>
